prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,14 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+console.log(languages);
+var declareFirstElement = 'The first element of the array is';
+var firstElement = 'html';
+console.log(declareFirstElement, firstElement);
+var declareThirdElement = 'The 3rd element of the array is';
+var thirdElement = 'javascript';
+console.log(declareThirdElement, thirdElement);
+var declareLength = 'The length of the array is';
+var length = 6;
```

Comment:
Here you want to use the `length` property of the `languages` array to get the `length` of the array.